### ISSUE TITLE
Prepending variables with export does not work.

### ISSUE THREAD

**Issue by einnjo** [sneakertack/nvar#4]
Node version: 8.10
Nvar version: 1.3.0

Assume this index.js for the following test cases:
```js
// index.js
const config = {};
require('nvar')({ target: config });
console.log(config);
```
Expected output for all of the following cases:
```js
{  foo: 'bar', hello: 'world' }
```
### No export:
Given:
```bash
#.env
foo=bar
hello=world
```
Received:
```js
{  foo: 'bar', hello: 'world' } // SUCCESS
```

### Export:
Given:
```bash
# .env
export foo=bar
export hello=world
```

Received:
```js
{ foo: 'bar' } // FAIL
```

**Comment by sneakertack** [comment]
:man_facepalming: @juanjoLenero you are right, can't believe that this slipped through. All the tests so far were done using singular `export` statements, but the library would fail from the 2nd `export` statement onwards.

I've fixed it (https://github.com/sneakertack/nvar/commit/6f9610e93ef79a3a4e24cd336fd4478ffce58b5e) and published as [v1.3.1](https://github.com/sneakertack/nvar/releases/tag/v1.3.1) - try it now. Thanks for taking the time to report the issue (solid reproduction steps), and apologies for the inconvenience caused.

**Comment by sneakertack** [comment]
Closing, fixed by [v1.3.1](https://github.com/sneakertack/nvar/releases/tag/v1.3.1).
